app/models/user.rb

```diff
@@ -415,6 +415,23 @@ def coc_codes_for_consent
     ConsentLimit.available_coc_codes
   end
 
+  def report_filter_visible?(key)
+    return true if can_view_project_related_filters?
+
+    project_related = [
+      :project_ids,
+      :organization_ids,
+      :data_source_ids,
+      :funder_ids,
+      :projects,
+      :organizations,
+      :data_sources,
+      :funding_sources,
```

Comment:
Do you need these? If not, not having aliases seems desirable?

Reply:
Yes, the control sections don't align with the field names 😞